topo_processor/metadata/metadata_validators/tests/metadata_validator_stac_test.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+import pytest
+
+from topo_processor.metadata.metadata_validators.metadata_validator_stac import MetadataValidatorStac
+from topo_processor.stac import Asset, Item
+
+
+@pytest.mark.asyncio
+async def test_check_validity():
+    source_path = os.path.join(os.getcwd(), "test_data", "tiffs", "SURVEY_1", "CONTROL.tiff")
+    asset = Asset(source_path)
+    item = Item("item_id")
+    item.add_asset(asset)
+    item.properties.update({"camera:nominal_focal_length": "string"})
+    item.properties.update({"camera:sequence_number": 1234})
+    item.add_extension("https://linz.github.io/stac/v0.0.2/camera/schema.json")
```

Comment:
Move extension version numbers to their own file so that we only need to bump version numbers in one location.

Reply:
I am unsure if we can bump all the versions of  the linz extensions at the same time, but we could have a `StacCameraExtension` object which that includes the specific version/uri we are using

Reply:
Yea we were discussing one version number for each extension, including community/core extensions. Risky having hard coded extension numbers in different parts of the code/tests and having to make sure you've found them all each time one of them needs to be bumped.

Reply:
And yes URI might be better than version number so that they could be swapped out e.g. when a "LINZ extension" becomes a "community extension".